css/css-color/xyz-003-ref.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 4: CSS Color 4: xyz</title>
+<style>
+    body { background-color: grey; }
+    .test { background-color: lab(100% 6.1097 -13.2268); width: 12em; height: 12em; } /* color(xyz 1 1 1) converted to Lab */
```

Comment:
This is exactly on the border of being an HDR color: same luminance as media white, but non-zero chroma so out of gamut for any non-HDR colorspace. Was that the intent?

Reply:
I wish I could say I was being clever, but I just picked `1 1 1` as it seemed nice.